assets/js/blocks/product-collection/inspector-controls/taxonomy-controls/taxonomy-item.tsx

```diff
@@ -0,0 +1,257 @@
+/**
+ * External dependencies
+ */
+import { useEntityRecords } from '@wordpress/core-data';
+import { Taxonomy } from '@wordpress/core-data/src/entity-types';
+import { useState, useMemo, useRef } from '@wordpress/element';
+import { useDebounce } from '@wordpress/compose';
+import { FormTokenField } from '@wordpress/components';
+
+// A constant empty array that is reused throughout the component.
+const EMPTY_ARRAY: [] = [];
+
+// Base arguments for querying terms.
+const BASE_QUERY_ARGS = {
+	order: 'asc',
+	_fields: 'id,name,slug',
+	context: 'view',
+};
+
+type Term = {
+	id: number;
+	name: string;
+	slug: string;
+};
+
+interface TaxonomyItemProps {
+	taxonomy: Taxonomy;
+	termIds: number[];
+	onChange: ( termIds: number[] ) => void;
+}
+
+// Function to get the term id based on user input in the `FormTokenField`.
+export const getTermIdByTermValue = (
```

Comment:
Is this function exported for some reason?

Reply:
My apologies. I accidentally left this function exported while moving it between files. There is no reason for this function to be exported. 😅